src/main/php/CommandLine/Library/GitInputValidator.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Zooroyal\CodingStandard\CommandLine\Library;
+
+use Symfony\Component\Process\Exception\ProcessFailedException;
+
+class GitInputValidator
+{
+    /** @var ProcessRunner */
+    private $processRunner;
+
+    /**
+     * GitInputValidator constructor.
+     *
+     * @param ProcessRunner $processRunner
+     */
+    public function __construct(ProcessRunner $processRunner)
+    {
+        $this->processRunner = $processRunner;
+    }
+
+    public function isCommitishValid(string $commitish)
```

Comment:
Function comment please

```suggestion
    public function isCommitishValid(string $commitish) : bool
```